aviary/utils/process_input_decks.py

```diff
@@ -164,10 +166,56 @@ def update_dependent_options(aircraft_values: AviaryValues(), dependent_options)
 
 def initial_guessing(aircraft_values: AviaryValues()):
     problem_type = aircraft_values.get_val('problem_type')
+    reserves_option = aircraft_values.get_val(Aircraft.Design.RESERVES_OPTION)
     reserves = aircraft_values.get_val(
-        Aircraft.Design.RESERVES) if initial_guesses['reserves'] == 0 else initial_guesses['reserves']
+        Aircraft.Design.RESERVES, units='lbm') if initial_guesses['reserves'] == 0 else initial_guesses['reserves']
+    if reserves_option == 1:
+        aircraft_values.set_val(Aircraft.Design.RESERVES, reserves, units='lbm')
+        if reserves < 10:
+            raise ValueError('"aircraft:design:reserves" must be greater than 10 lbm.')
```

Comment:
Just to be safe in case the variable name changes, I would use the actual variable name of Aircraft.Design.RESERVES here in the message instead of typing out its string name

Reply:
Done.

Reply:
Can you update the names to make it clear they refer to fuel? Such as FIXED_RESERVE_FUEL and RESERVE_FUEL_FRACTION

Reply:
Removed the check.